tests/test-pwm.js

```diff
@@ -54,7 +54,7 @@ expectThrow("pwmpin: set pulseWidth without period", function () {
     pinA.setPulseWidth(300);
 });
 
-pinA = pwm.open({channel: pins.IO3, period: 3, pulseWidth: 1});
+pinA = pwm.open({ channel: pins.IO3, period: 3, pulseWidth: 1 });
 assert(pinA != null && typeof pinA == "object",
```

Comment:
use !== and ===, this is not related to this patch, but you might want to fix these here.